ush/retrieve_data.py

```diff
@@ -897,10 +954,17 @@ def parse_args(argv):
     # Required
     parser.add_argument(
         "--anl_or_fcst",
-        choices=("anl", "fcst"),
+        choices=("anl", "fcst", "obs", "None"),   # Added 'None' for generic packages copied from local disk or remote url
         help="Flag for whether analysis or forecast \
         files should be gathered",
-        required=True,
+        required=False,                    # relaxed this arg option, to enable generic package copying
+    )
+    parser.add_argument(
+        "--machine",
+        choices=("jet", "hera"),  
+        help="Flag for which machine path(s) to use \
+        if retrieving local data files", 
+        required=False,                    # relaxed this arg option, to enable generic package copying
```

Comment:
Please do not incorporate machines into this tool. That should all be handled by the App.

Reply:
removed all machine references